client/App.jsx

```diff
@@ -2,40 +2,104 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Router, Route, Link } from 'react-router'
 import Login from './Login.jsx'
-import TaskView from './TaskView.jsx'
+
+import Task from './Task.jsx';
+import Completed from './Completed.jsx';
+
+
+/* Test Data inserted here */
+var data = require('./fakeData');
+// Somehow determines the 5 or fewer most urgent tasks to be complete
+var urgentTasks = data.allTasks;//.slice(0, 2);
+var completedTasks = data.allCompletedTasks;
+
+
+// Necessary for Material-UI
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+
+// Necessary for simple Mobile/Web click functionality on components
+import injectTapEventPlugin from 'react-tap-event-plugin';
+injectTapEventPlugin();
+
+
+// Components we have built
+import Navbar from './Navbar.jsx';
+import Circle from './Circle.jsx';
+import Comp from './Comp.jsx';
+
+
+//import Avatar from 'material-ui/Avatar';
+//import Chip from 'material-ui/Chip';
+//import FontIcon from 'material-ui/FontIcon';
+
+//import AppBar from 'material-ui/AppBar';
+
+let test = () => {
+  console.log('test click worked');
+}
 
 class App extends React.Component {
   constructor() {
     super();
+
     this.state = {
-      allTasks: [
-        'dishes',
-        'walk dog',
-        'laundry',
-        'other stuff'
-      ]
-    };
+      urgentTasks: urgentTasks,
+      completedTasks: completedTasks
+    }
   }
 
   render() {
     return (
-      <div>
-        <div>
-          <ul>
-            <li><Link to="/login">Login</Link></li>
-          </ul>
-        </div>
+      <MuiThemeProvider>
         <div>
-          <TaskView tasks={this.state.allTasks}/>
+          <Navbar />
+
+          <div className="row">
+            <div className="col-xs-10">
+              {/* Create the urgentTask bubbles */}
+              {this.state.urgentTasks.map((urgentTask, i) => {
+                let urgentTaskName = urgentTask.name;
+                let urgentTaskDueBy = urgentTask.dueBy.toString();
+                console.log(urgentTaskDueBy);
+                return (
+                  <div className="col-xs-3">
+                    <Circle name={urgentTaskName} dueBy={urgentTaskDueBy} key={i}/>
+                  </div>
```

Comment:
Normally yes but React doesnt allow me to pass objects as children, so I have to expose the properties on the object first.